mitodl/mit-learn
# remove learning_path from search

### What are the relevant tickets?
None

### Description (What does it do?)
This pr removes learning paths from the search results. Currently we do not want learning paths to be displayed in the search index. We asked people to set any learning paths they create as private but we should just not index them regardless of setting

### Screenshots (if appropriate):
None

### How can this be tested?
In the main branch create a public learning path if you don't already have any in the index.
Go to http://open.odl.local:8062/search?resource_category=learning_material and verify that "learning path" is a facet

Switch to this branch
Go to http://open.odl.local:8062/search?resource_category=learning_material and verify that "learning path" is  not a facet

Create a new public learning path and verify that there isn't an error
